types/storybook__addon-storyshots/index.d.ts

```diff
@@ -5,18 +5,23 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.1
 
-import { IStorybookStory } from '@storybook/react';
+import { getStorybook } from '@storybook/react';
```

Comment:
`IStorybookStory` is available from the types file:

```suggestion
import { IStorybookStory } from '@storybook/react/dist/client/preview/types';
```